examples/tutorials/torch_c_example.c

```
@@ -0,0 +1,66 @@
+    Optimizer* opt = torch_optim_adam((Module*)model, 0.05f, 0.0f, 0.9f, 0.999f, 1e-8f);
+
+    for (int epoch = 1; epoch <= 100; epoch++) {
+        Tensor* pred = torch_nn_sequential_forward(model, train->X);
+        Tensor* loss = torch_nn_mse_loss(pred, train->y);
+
+        torch_optim_zero_grad(opt);
+        torch_backward(loss, NULL, false, false);
+        torch_optim_step(opt);
+        torch_reset_ir();
+
+        if (epoch % 20 == 0 || epoch == 1)
```

Cppcheck (2.21.0)

[style] The function 'torch_tensor_device_fast' is never used.

(unusedFunction)